src/com/TTT/Board.java

```diff
@@ -7,20 +7,25 @@
 
     private String[] spaces;
 
-    public Board() {
-        spaces = new String[]{"1","2","3","4","5","6","7","8","9"};
+    public Board(int size) {
+        if(size == 4) {
+            spaces = new String[]{"1", "2", "3", "4", "5", "6", "7", "8", "9", "10", "11", "12", "13", "14", "15", "16"};
+        }
+        else {
+            spaces = new String[]{"1","2","3","4","5","6","7","8","9"};
+        }
```

Comment:
You could initialise using a loop? That way if we introduced 5x5, you wouldnt have to make any more changes here.